Usage/faq.md

```diff
@@ -132,6 +134,19 @@ If you experience performance issues and cannot identify the cause, or suspect a
 
 We recommend first testing with all browser extensions and third-party SillyTavern extensions disabled to isolate the source of the performance degradation.
 
+### When I import a lot of characters, the app becomes slow. Why?
+
+Unfortunately, SillyTavern wasn't designed to handle huge character libraries. The more you have, the longer it will take to load the character list. Evidential data suggests that the performance degradation starts to become noticeable when you have more than 1000 characters.
+
+However, there are some things you can do to mitigate the issue:
+
+1. Enable lazy loading of characters setting the value `performance.lazyLoadCharacters` to true in the `config.yaml` file. After the next server restart, the character list will only load the full data of characters you interact with. Please be aware that some third-party extensions may not work correctly with this setting enabled if they were not updated to support it (contact the extension developer for more information).
```

Comment:
Should we mention here that this will also lead to inconsistent results with advanced search, that can't search char definitions until they are loaded?